edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/NestedScopeRequest.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// This is an object to encapsulate JSON serialization of the
+    /// GetDevicesAndModulesInTargetScope POST request.
+    /// </summary>
+    class NestedScopeRequest
+    {
+        [JsonProperty(PropertyName = "pageSize")]
+        public int PageSize { get; set; }
+
+        [JsonProperty(PropertyName = "continuationLink", NullValueHandling = NullValueHandling.Ignore)]
+        public string ContinuationLink;
```

Comment:
nit: ContinuationToken?

Reply:
yea "Link" is technically incorrect terminology for POSTs.  But service side names it ContinuationLink due to legacy, as do our existing request/response classes, so I kept it as such for consistency.